halo/memory/gamestate/player.cpp

```diff
@@ -5,3 +5,13 @@
  */
 
 #include "player.hpp"
+#include "../../../hooker/hooker.hpp"
+
+Signature(true, sig_player_table,
+    {-1, -1, -1, 0x00, 0x89, 0x44, 0x24, 0x48, 0x35, 0x72, 0x65, 0x74, 0x69});
+
+PlayerTable* player_table(){
+    static PlayerTable** player_table_ptr =
```

Comment:
It looks like you want to cache this value so it's only created once, but you're re-assigning it every time this function is run. From our conversation, this pattern shows up a few times, so I've opened #47 to deal with this